ReactAndroid/src/main/java/com/facebook/react/bridge/Arguments.java

```diff
@@ -277,7 +315,57 @@ public static WritableMap fromBundle(Bundle bundle) {
   }
 
   /**
+   * Convert a {@link WritableArray} to a {@link ArrayList}.
+   *
+   * @param readableArray the {@link WritableArray} to convert.
+   * @return the converted {@link ArrayList}.
+   */
+  @Nullable
+  public static ArrayList toList(@Nullable ReadableArray readableArray) {
```

Comment:
The reason why I use `ArrayList` is because it is `Serializable`, while a `List` isn't, so I can use it in `toBundle` with `Bundle.putSerializable`.

Do you think it's better to return `List` and then cast it in `toBundle`?